Makefile

```diff
@@ -0,0 +1,20 @@
+MODULE = gammapy
+
+CURDIR = $(shell pwd)
+
+build:
```

Comment:
I have removed some targets: `build`, `doc`, `test`, since you mentioned it is not useful to have targets that are wrappers to `setup.py`. We can add them back. I think it'd be nice to have a few common ones because:
1. `TAB` completion for targets works nice with makefiles in not too old shells, but it doesn't with setup.py.
2. we can define an `all` target to run `develop`, `test` and `doc` at the same time (similar to what I had before).
What shall we do?